src/main/resources/root.jsp

```diff
@@ -95,18 +95,24 @@
         }
 
         .jahia-loader {
-            height: 100vh;
             width: 100vw;
+            height: 100vh;
+            background-color: #1f262a;
```

Comment:
any specific reasons to change the color here?
They actually seem very similar to me 😅

Reply:
yes, to match the one that is loaded by moonstone afterwards